js/src/components/01_atoms/DownloadCourse/index.js

```diff
@@ -109,7 +109,7 @@ class DownloadCourse extends React.Component {
       urlsToCache.push(`${getLangCodePrefix()}/courses`);
 
       // Cache Course data.
-      urlsToCache.push(`${getLangCodePrefix()}/node/${course.id}`);
```

Comment:
1. Instead of using custom getLangCodePrefix() method, please do use `Drupal.url()` instead. It adds language prefix automatically if it exists.
2. All nodes have full URL property, so using course.path is likely redundant - you should use course.url instead (same for other content types).